components/auth/GoogleAuth.tsx

```
@@ -1,76 +1,89 @@
+      size={GoogleSigninButton.Size.Wide}
+      color={GoogleSigninButton.Color.Dark}
+      onPress={async () => {
+        try {
+          await GoogleSignin.hasPlayServices();
+          const response = await GoogleSignin.signIn();
+          if (isSuccessResponse(response)) {
+            const { idToken, user } = response.data;
+
+            const { data, error } = await supabase.auth.signInWithIdToken({
+              provider: "google",
+              token: idToken!!,
```

Biome (1.9.4)

[error] Forbidden extra non-null assertion.

Safe fix: Remove extra non-null assertion.


(lint/suspicious/noExtraNonNullAssertion)